src/identity/actions/creators/index.ts

```diff
@@ -0,0 +1,46 @@
+import {CurrentIdentity, CurrentOrg} from 'src/identity/apis/auth'
+import {RemoteDataState} from 'src/types'
+
+export const SET_QUARTZ_IDENTITY = 'SET_QUARTZ_IDENTITY'
+export const CLEAR_QUARTZ_IDENTITY = 'CLEAR_QUARTZ_IDENTITY'
+export const SET_QUARTZ_IDENTITY_STATUS = 'SET_QUARTZ_IDENTITY_STATUS'
+export const SET_CURRENT_BILLING_PROVIDER = 'SET_CURRENT_BILLING_PROVIDER'
+export const SET_CURRENT_ORG_DETAILS = 'SET_CURRENT_ORG_DETAILS'
+
+export type Actions =
+  | ReturnType<typeof setQuartzIdentity>
+  | ReturnType<typeof clearQuartzIdentity>
+  | ReturnType<typeof setQuartzIdentityStatus>
+  | ReturnType<typeof setCurrentBillingProvider>
+  | ReturnType<typeof setCurrentOrgDetails>
+
+export const setQuartzIdentity = (identity: CurrentIdentity) =>
+  ({
+    type: SET_QUARTZ_IDENTITY,
+    identity: identity,
+  } as const)
```

Comment:
[YAGNI](https://martinfowler.com/bliki/Yagni.html) - I know we talked about this, but don't add this until we need to use it. You won't forget the concept by the time we need to clear out the quartz identity

Reply:
We need `SET_QUARTZ_IDENTITY`, which is what gets used by the `quartzIdentity` thunk, but I'll remove `CLEAR_QUARTZ_IDENTITY`, which I added later, and which you're right we're not using at all yet.